contrib/pyln-client/pyln/client/__init__.py

```diff
@@ -2,7 +2,7 @@
 from .plugin import Plugin, monkey_patch
 
 
-__version__ = "0.8.0"
+__version__ = "0.9.0"
```

Comment:
Hmm why did you bump the version ?

Reply:
Hmm, good question. Because my understanding was that the tests used `pylightning` which used `pyln-client` which I extended with a new argument to `close()` and so `pylightning` and tests would not work with an earlier version of `pyln-client`, so I bumped the version with the intent to require `pyln-client>=0.9.0` in `pylightning`.

But now I double checked the relationship and I see that tests use `pyln-client` [directly](https://github.com/ElementsProject/lightning/blob/15ca3b6/Makefile#L275) and that `pylightning` is an independent package seemingly not used by anybody.

Reverted the above change and the change to `contrib/pylightning/requirements.txt` which was supposed to require the new version with the extended `close()`.

Reply:
Yeah we explicitly set the `PYTHONPATH` to test against the newest `pyln`, and bump the version at each release.